common/api/core-common.api.md

```diff
@@ -2021,6 +2092,8 @@ export class DisplayStyle3dSettings extends DisplayStyleSettings {
 // @public
 export interface DisplayStyle3dSettingsProps extends DisplayStyleSettingsProps {
     ao?: AmbientOcclusion.Props;
+    // (undocumented)
+    atmosphericScattering?: AtmosphericScattering.Props;

```

Comment:
This should not be a view flag. It should probably be a boolean `Environment.displayAtmosphere` like `displaySky` and `displayGround`.